dll/cpl/input/lang/he-IL.rc

```diff
@@ -7,14 +7,18 @@ STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
 CAPTION "הגדרות"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    LTEXT "Select the services that you want for each input language shown in the list.\nUse the Add and Remove buttons to modify this list.", -1, 9, 6, 238, 25
-    CONTROL "TEXT", IDC_KEYLAYOUT_LIST, "SYSLISTVIEW32", WS_BORDER | WS_VSCROLL | WS_TABSTOP | LVS_SINGLESEL | LVS_NOSORTHEADER | LVS_SHOWSELALWAYS | LVS_REPORT, 8, 36, 237, 101
-    PUSHBUTTON "קבע כברירת מחדל...", IDC_SET_DEFAULT, 101, 159, 144, 14
-    PUSHBUTTON "הוסף...", IDC_ADD_BUTTON, 27, 142, 70, 14
-    PUSHBUTTON "הסר", IDC_REMOVE_BUTTON, 101, 142, 70, 14
-    PUSHBUTTON "מאפיניים...", IDC_PROP_BUTTON, 175, 142, 70, 14
-    GROUPBOX "העדפות", -1, 7, 185, 240, 36
-    PUSHBUTTON "הגדרות מקשים...", IDC_KEY_SET_BTN, 14, 198, 110, 14
+    GROUPBOX "Default input &language", -1, 5, 5, 246, 55
+    LTEXT "Select one of the installed input languages to use when your computer starts.", -1, 15, 15, 230, 23
+    COMBOBOX IDC_DEFAULT_LANGUAGE, 15, 40, 230, 300, CBS_HASSTRINGS | CBS_AUTOHSCROLL | CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
+    GROUPBOX "&Installed services", -1, 5, 65, 246, 125
+    LTEXT "Select the services that you want for each input language shown in the list. Use the Add and Remove buttons to modify this list.", -1, 15, 78, 230, 25
+    CONTROL "", IDC_KEYLAYOUT_LIST, "SysTreeView32", TVS_SHOWSELALWAYS | WS_BORDER | WS_VSCROLL | WS_TABSTOP, 15, 105, 155, 80
+    PUSHBUTTON "הוסף...", IDC_ADD_BUTTON, 175, 105, 70, 14
+    PUSHBUTTON "הסר", IDC_REMOVE_BUTTON, 175, 125, 70, 14
+    PUSHBUTTON "מאפיניים...", IDC_PROP_BUTTON, 175, 145, 70, 14
+    GROUPBOX "העדפות", -1, 5, 195, 245, 30
```

Comment:
For some reason the groupbox line here doesn't display correctly.

Reply:
It's a Right-to-Left text.

Reply:
Sure, except that why does the problem show there only here and not for the other strings above?

Reply:
https://www.compart.com/en/unicode/U+05D4
The character "ה" is a right-to-left letter.

Reply:
But aren't they **all** RTL ? :smile:

Reply:
Umm... Dunno.